src/main/java/core/basesyntax/DrawFigure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface DrawFigure {
```

Comment:
According to Java naming convention, interface name should be an adjective, may also be a noun, but not verb.